src/main/resources/public/src/views/DataRequest.vue

```diff
@@ -71,25 +80,48 @@
                     this.dataRequest = response.data
                     //Even if typeahead value = dataRequest.model - we need to fullfill typeahead inputValue to display value
                     this.$refs.typeahead.$data.inputValue = this.dataRequest.model
+                    this.$refs.clamercollectivity.$data.inputValue = this.dataRequest.organization
                   })
                   .catch(e => {
                     this.errors.push(e)
                   })
             }
 
+            //Get first hundred models from datacore
             axios.get('/api/data_access_request/123456789/model')
                 .then(response => {
                     this.models = response.data
                 })
                 .catch(e => {
                     this.errors.push(e)
                 })
+
+            //Get first hundred organization name from datacore
+            axios.get('/api/data_access_request/organisation?name=')
+                .then(response => {
+                    response.data.forEach( data => {
+                        this.organizations.push(data.denominationUniteLegale)
```

Comment:
I wanted to keep organization uri as string with only legal name as input v-model but it took me some times. It is done now in the last commit. It can be improved, for example with saving organization as Organization object in the mongo document, but it will need some back-end additionnal modifications, plus discussing if this additionnal amount of data really need to be saved in this data access request document.